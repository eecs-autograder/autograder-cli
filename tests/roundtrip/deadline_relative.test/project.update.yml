project:
  name: Test Project
  timezone: America/Chicago
  course:
    name: Test Course
    semester: Summer
    year: 2014
  settings:
    deadline:
      cutoff_type: relative
      deadline: Feb 9, 2021 6:45am
      cutoff: 42m
