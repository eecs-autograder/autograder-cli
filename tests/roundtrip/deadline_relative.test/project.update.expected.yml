project:
  name: Test Project
  timezone: America/Chicago
  course:
    name: Test Course
    semester: Summer
    year: 2014
  settings:
    deadline:
      cutoff_type: relative
      deadline: Feb 09, 2021 06:45AM
      cutoff: 42m
